user_guide/configuration/feature_suite_and_scenario_configuration.rst

```diff
@@ -1,10 +1,10 @@
-Configuring Test Suites
-=======================
+Feature, Suite and Scenario Configuration
```

Comment:
The title is weord. this chapter is not configuring features and scenarios (they cannot be configured)